config/features.yml

```diff
@@ -316,6 +316,10 @@ features:
     actor_type: user
     description: When enabled, claims status tool will use the new 5103 alert designs and hides the ask your claim decision section
     enable_in_development: true
+  cst_show_document_upload_status:
+    actor_type: user
+    description: When enabled, the frontend will be able to view the document upload status that comes from the evidence_submissions table.
```

Comment:
Can we change this to `When enabled, claims status tool will display the upload status that...`?